src/Tag/Script.php

```diff
@@ -32,6 +35,31 @@ public function getContent(): string
         return $this->content;
     }
 
+    /**
+     * @link https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/nonce
+     */
+    public function nonce(bool|string $nonce): self
```

Comment:
Maybe split this method to `nonce(?string $nonce)` and `generateNonce(?string $prefix = null)`?